docsets/Heron/docset.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Heron",
+    "version": "0.14.0",
+    "archive": "Heron.tgz",
```

Comment:
You need to add this archive as well. It's not enough to have it in the `versions/` folder only.